src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -447,6 +447,135 @@ JSON example:
+| LabelingType                      | REQUIRED                                                                                                                        | [string][]                              | `CASL` , `PCASL` , `PASL`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| PostLabelingDelay                 | REQUIRED                                                                                                                        | [number][] or [array][] of [numbers][]  | This is the time, in seconds, after the the end of the labeling (for `(P)CASL`) or middle of the labeling pulse (for `PASL`) until the middle of the excitation pulse applied to the imaging slab (for 3D acquisition) or first slice (for 2D acquisition).  Can be a number (for a single-PLD time series) or an array of numbers (for multi-PLD and Look-Locker). In the latter case, the array of numbers contains the PLD of each volume, namely each `control` and `label`, in the acquisition order. Any image within the time-series without a PLD, for example an `m0scan`, is indicated by a zero. Based on DICOM Tags 0018,9079 `Inversion Times` and 0018,0082 `InversionTime`.                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
Define PLD acronym, is it PostLabelingDelay? I think it's been never defined.
Also a summary figure like in the PET extension may help to visualize some parameters

Reply:
Yes thanks!